src/olympia/stats/tests/test_cron.py

```diff
@@ -27,7 +27,7 @@ def test_stats_for_date(self):
 
     def test_count_stats_for_date(self):
         # Add a listed add-on, it should show up in "addon_count_new".
-        listed_addon = addon_factory()
+        listed_addon = addon_factory(created=datetime.datetime.now())
```

Comment:
fun fact - the support for randomly assigning `created` dates in `addon_factory` never worked until now (cue hours of head scratching why `addon_factory` was creating add-ons with dates from 2011).

Reply:
Oh wait, this is affected by using `auto_now_add`? If so, this is confusing and needs a comment and somehow even feels like a Django bug to me :thinking:

Reply:
well, yeah, it's affected because the support from choosing a random date in addon_factory never worked (see notes on `auto_now_add` you linked to).  Now it does work so the one test class that relied on created dates from addon_factory being today's date failed.  Where would suggest documenting it?  

> somehow even feels like a Django bug to me 🤔

I dunno - the documentation is pretty explicit about how `auto_now_add` works so I can't see it's a bug.

Reply:
Yeah, nevermind, I didn't read the docs properly and kinda forgot over the ages that I continuously ran into that problem too :man_facepalming: